src/app/components/Search/SearchSuggestionsLinks.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC } from 'react'
+import Box from '@mui/material/Box'
+import { searchSuggestionTerms } from '../../components/Search/search-utils'
+import { Trans, useTranslation } from 'react-i18next'
+import { Link as RouterLink } from 'react-router-dom'
+import Link from '@mui/material/Link'
+import { RouteUtils } from '../../utils/route-utils'
+import { OptionalBreak } from '../OptionalBreak'
+
+interface Props {}
+
+export const SearchSuggestionsLinks: FC<Props> = () => {
+  const { t } = useTranslation()
+  const { suggestedBlock, suggestedTransaction, suggestedAccount } = searchSuggestionTerms
+
+  return (
+    <Trans
+      t={t}
+      i18nKey="search.searchSuggestions"
+      components={{
+        OptionalBreak: <OptionalBreak />,
+        BlockIcon: <></>,
+        BlockLink: <Link component={RouterLink} to={RouteUtils.getSearchRoute(suggestedBlock)} />,
```

Comment:
Slightly cheated here: these suggestions redirect to search results, instead of merely modify search input like SearchSuggestionsButtons. I didn't want to maintain emitting an event out from suggestions and drilling setValue into instances of Search.

Reply:
Don confirms THIS IS BETTER